tests/integration/targets/s3_bucket/roles/s3_bucket/tasks/public_access.yml

```diff
@@ -0,0 +1,77 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+
+    # ============================================================
+
+    - name: 'Create a simple bucket with public access block configuration'
+      s3_bucket:
+        name: '{{ bucket_name }}'
+        state: present
+        public_access:
```

Comment:
Please also add a test to the 'basic' suite to make sure your PR doesn't change the default behaviour.

Reply:
I don't think I get this. Can you please explain in further detail? I don't see any file named basic in tests/. I've followed what's already in place for 'encryption_kms' test. It's in inventory and in tasks/. Thanks in advance.

Reply:
Simplest version would be to tweak https://github.com/ansible-collections/amazon.aws/blob/main/tests/integration/targets/s3_bucket/roles/s3_bucket/tasks/simple.yml and make sure that the returned values for public_access match those of Amazon's defaults if you didn't pass public_access.

The intention is to ensure that in adding the new public_access option you've not changed the default behaviour when someone creates a bucket.

Reply:
Ok, makes perfect sense now:) TY.